scripts/src/java/org/oppia/android/scripts/ScriptResultConstants.kt

```diff
@@ -5,4 +5,8 @@ object ScriptResultConstants {
   /** RegexPatternValidationCheck output message constants. **/
   val REGEX_CHECKS_PASSED: String = "REGEX PATTERN CHECKS PASSED"
   val REGEX_CHECKS_FAILED: String = "REGEX PATTERN CHECKS FAILED"
+
+  /** XMLSyntaxCheck output message constants **/
+  val XML_SYNTAX_CHECK_PASSED: String = "XML SYNTAX CHECK PASSED"
```

Comment:
Maybe ``XML_SYNTAX_CHECK_PASSED_OUTPUT_INDICATOR`` might be a bit clearer on how this is being used. WDYT?

Ditto for the others.

Reply:
SGTM. DONE.